tests/unit/llm/test_openai_llm.py

```diff
@@ -13,16 +13,18 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 from unittest.mock import MagicMock, Mock, patch
+from typing import Any, cast
 
 import openai
 import pytest
 from neo4j_graphrag.exceptions import LLMGenerationError
 from neo4j_graphrag.llm import LLMResponse
 from neo4j_graphrag.llm.openai_llm import AzureOpenAILLM, OpenAILLM
+from neo4j_graphrag.llm.types import ToolCallResponse
 
 
-def get_mock_openai() -> MagicMock:
-    mock = MagicMock()
+def get_mock_openai() -> Any:
+    mock = cast(Any, MagicMock())
```

Comment:
That's so... mypy-ugly :) I hope there is another way to make mypy happy but I need to think more about it :)